website/versioned_docs/version-v1.x/trusted-setup.md

```diff
@@ -19,28 +19,58 @@ trusted setup. After all, MACI's security model presumes a trusted coordinator.
+| processMessagesNonQv vKey              | The verification key that can be used to verify the processMessagesNonQv circuit's proofs                | 6-9-2-3    | [maci-processmessagesnonqv_6-9-2-3_vkey.json](https://maci-v1-2-0-trusted-setup-ceremony-pse-p0tion-production.s3.eu-central-1.amazonaws.com/circuits/maci-processmessagesnonqv_6-9-2-3/maci-processmessagesnonqv_6-9-2-3_vkey.json)            |
+| tallyVotes vKey                        | The verification key that can be used to verify the tallyVotes circuit's proofs                          | 6-2-3      | [tallyvotes_6-2-3_vkey.json](https://maci-v1-trusted-setup-ceremony-pse-p0tion-production.s3.eu-central-1.amazonaws.com/circuits/tallyvotes_6-2-3/tallyvotes_6-2-3_vkey.json)                                                                   |
+| tallyVotesNonQv vKey                   | The verification key that can be used to verify the tallyVotesNonQv circuit's proofs                     | 6-2-3      | [tallyvotesnonqv_6-2-3_vkey.json](https://maci-v1-2-0-trusted-setup-ceremony-pse-p0tion-production.s3.eu-central-1.amazonaws.com/circuits/maci-tallyvotesnonqv_6-2-3/maci-tallyvotesnonqv_6-2-3_vkey.json)                                      |
+| processMessages Solidity verifier      | The Solidity smart contract which can be used to verify the processMessages circuit proofs on-chain      | 6-9-2-3    | [maci-processmessages_6-9-2-3_verifier.sol](https://maci-v1-2-0-trusted-setup-ceremony-pse-p0tion-production.s3.eu-central-1.amazonaws.com/circuits/maci-processmessages_6-9-2-3/maci-processmessages_6-9-2-3_verifier.sol)                     |
+| processMessagesNonQv Solidity verifier | The Solidity smart contract which can be used to verify the processMessagesNonQv circuit proofs on-chain | 6-9-2-3    | [maci-processmessagesnonqv_6-9-2-3_verifier.sol](https://maci-v1-2-0-trusted-setup-ceremony-pse-p0tion-production.s3.eu-central-1.amazonaws.com/circuits/maci-processmessagesnonqv_6-9-2-3/maci-processmessagesnonqv_6-9-2-3_verifier.sol)      |
+| tallyVotes Solidity verifier           | The Solidity smart contract which can be used to verify the tallyVotes circuit proofs on-chain           | 6-2-3      | [tallyvotes_6-2-3_verifier.sol](https://maci-v1-trusted-setup-ceremony-pse-p0tion-production.s3.eu-central-1.amazonaws.com/circuits/tallyvotes_6-2-3/tallyvotes_6-2-3_verifier.sol)                                                             |
+| tallyVotesNonQv Solidity verifier      | The Solidity smart contract which can be used to verify the tallyVotesNonQv circuit proofs on-chain      | 6-2-3      | [tallyvotesnonqv_6-2-3_verifier.sol](https://maci-v1-2-0-trusted-setup-ceremony-pse-p0tion-production.s3.eu-central-1.amazonaws.com/circuits/maci-tallyvotesnonqv_6-2-3/maci-tallyvotesnonqv_6-2-3_verifier.sol)                                |
+
+## Maci v1.1 Trusted Setup Ceremony
```

Comment:
don't know why on my computer doesn't display an H2 title here (only after I add extra space between # and M --> which means it only works on `##  Maci v1.1 Trusted Setup Ceremony`)
if this just an issue on my computer, then ignore this comment :)

Reply:
yea I get that too, I removed space and added again, let's see if it works